server/apis/workspace/service.test.ts

```diff
@@ -91,4 +99,49 @@ describe('join', () => {
   });
 });
 
+describe('info', () => {
+  const WORKSPACE_ID = 1;
+  const INVALID_WORKSPACE_ID = -1;
+
+  it('워크스페이스 ID가 DB에 존재할 경우 조회에 성공한다.', async () => {
+    const WORKSPACE_NAME = 'Wab';
+
+    workspaceModel.findOne.mockResolvedValueOnce({
+      id: WORKSPACE_ID,
+      name: WORKSPACE_NAME,
+      code: VALID_CODE,
+      users: [],
+      moms: [],
+    });
+
+    expect(workspaceService.info(WORKSPACE_ID)).resolves.toEqual({
+      name: WORKSPACE_NAME,
+      users: [],
+      moms: [],
+    });
+  });
```

Comment:
원희쌤한테 배웠는데 이렇게 변수를 따로 만들어 놓고 사용하면 이해하기 편하더라구요!
https://github.com/boostcampwm-2022/web27-Wabinar/blob/36ff386ef7cee33a3a0d0f92b0e4f19fc13c3d74/server/apis/user/service.test.ts#L15-L16

```suggestion
const successfulWorkspace = {  ... }

expect(...).resolves.toEqual(successfulWorkspace);
```

Reply:
아 저거 User interface를 가져와야 하는데 `require('@apis/user/model')` 여기있는 User 가져오려니까 undefind뜨고 안돼서 그냥 []로 두자.. 했어요.. 
아마도 @apis/user/model 여기서 default로 export 하는것만 가져와 지는거 같아요 ..? 
타입만 export 하는 디렉토리를 하나 만들까요?

- 이런식으로 선언하려고 했었어요
```ts
const USER_ARRAY: Pick<User, 'name' | 'avatarUrl'>[]
```

Reply:
일단 만들어서 해볼게요

Reply:
엥 잘못 이해한거 같애요 
```ts
workspaceModel.findOne.mockResolvedValueOnce({
      id: WORKSPACE_ID,
      name: WORKSPACE_NAME,
      code: VALID_CODE,
      users: [],
      moms: [],
    });
    expect(workspaceService.info(WORKSPACE_ID)).resolves.toEqual({
      name: WORKSPACE_NAME,
      users: [],
      moms: [],
    });
  });
```
이걸 하나의 변수로 만들라는 의미인가요 ..?

Reply:
헉 어디에서 interface가 필요하셨나요???

Reply:
아 삭제했어요 새로고침 한번 해보세요 ㅋㅋㅋㅋㅋㅋㅋ

Reply:
user에 있네요 한번 베껴볼게요

Reply:
하나는 제가 잘못봐서 하나만 한거에요 !!! ㅋㅋㅋㅋㅋ 저렇게 상수 빼놓고 사용하면 이름만보고 바로 확인이 가능해서 좋더라구요 !

Reply:
궁금한게 const successfulWorkspaces: Object[] = [];  왜 빈배열로 해주는건가요?
그 안에 들어있는 Object요소는 필요없나요?

Reply:
변수로 따로 두려니까 잘 안돼요..
- 변수로
![스크린샷 2022-11-16 오후 7 42 18](https://user-images.githubusercontent.com/63364990/202159304-7a400499-6395-4d34-83c1-aebcf5b567ae.png)
- 직접 넣은거
![스크린샷 2022-11-16 오후 7 42 22](https://user-images.githubusercontent.com/63364990/202159327-f17cff88-21ab-4e51-a8db-b62c40d6ca26.png)

Reply:
헉 그럼 일단 나두는걸로 해요!